ui-tests/src/test/java/org/openmrs/reference/security/RolesFieldTest.java

```diff
@@ -0,0 +1,64 @@
+package org.openmrs.reference.security;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
+import org.openmrs.reference.page.AddEditUserPage;
+import org.openmrs.reference.page.AdministrationPage;
+import org.openmrs.reference.page.HeaderPage;
+import org.openmrs.reference.page.HomePage;
+import org.openmrs.reference.page.ManageRolesPage;
+import org.openmrs.reference.page.ManageUserPage;
+import org.openmrs.reference.page.RoleFormPage;
+import org.openmrs.uitestframework.test.TestBase;
+
+public class RolesFieldTest extends TestBase {
+	
+	private HomePage homePage;
+	private HeaderPage headerPage;
+	private ManageRolesPage manageRolesPage;
+	private AdministrationPage administrationPage;
+	private RoleFormPage roleFormPage;
+	private ManageUserPage manageUserPage;
+	private AddEditUserPage addEditUserPage;
+	private static String user = "<script>alert(1);</script>";
+	private static String description = "some description";
```

Comment:
Generally, for XSS checks, we'd like to fill this into every text field, so we should at least have a test that makes the description of a role `<script>alert(1);</script>`. Similarly, we should test what happens if we have a privilege named `<script>alert(1);</script>` and we assign that to a role.